uint8 node_type
uint8 node_seq
uint8 msg_type
uint8[] payload